### DISCUSSION TITLE
About the unused effects in the effects folder

### DISCUSSION THREAD

**Discussion by LOuroboros** [Blinue/Magpie#309] [Q&A] [answered]
Hello Blinue!
I learned about this program of yours this very morning, Magpie. It's a freaking bless.
It's been doing wonders for RPG Maker based games on my end. It's exactly what I needed, much lighter and nicer to use than something like SandBurst.

Now, with that out of the way, I've a question to ask. I took a quick look at the files of Magpie, and I noticed there's a pretty beefy amount of effects that can not be selected from within Magpie's GUI by default. I suppose it's because they're not configured in the ScaleModels.json file?
For example, there's Pixellate, Linear, SSimDownscaler, SharpBilinear, and some more.

I was wondering; are these going to become usable at some point, or the user's supposed to set them up by themself?
As of right now, this is how things are looking on my end:
![explorer_11022022_0332967](https://user-images.githubusercontent.com/4485172/153546999-785498df-c340-4ffb-9ed3-35493352c558.png)

**Comment by bloc97** [comment]
You can already use them. I think they're not in the default list because they are tuned for very specific scenarios. For the vast majority of games, the default list works well enough. You can edit the default list in `ScaleModels.json` and apply any shader in any combination/order that you want.

**Comment by Blinue** [comment, marked as answer]
[Docs for ScaleModels.json](https://github.com/Blinue/Magpie/wiki/%E8%87%AA%E5%AE%9A%E4%B9%89%E7%BC%A9%E6%94%BE%E9%85%8D%E7%BD%AE) (Chinese)

Thanks to Z-Dante for translating it into English (a bit outdated though)
https://github.com/Z-Dante/Magpie/blob/master_v0.8.1/docs/CustomScaleConfiguration.md

Take a look at this as well
https://gist.github.com/hooke007/818ecc88f18e229bca743b7ae48947ad

**Reply by LOuroboros** [reply]
> [Docs for ScaleModels.json](https://github.com/Blinue/Magpie/wiki/%E8%87%AA%E5%AE%9A%E4%B9%89%E7%BC%A9%E6%94%BE%E9%85%8D%E7%BD%AE) (Chinese)
> 
> Thanks to Z-Dante for translating it into English (a bit outdated though) [Z-Dante/Magpie@`master_v0.8.1`/docs/CustomScaleConfiguration.md](https://github.com/Z-Dante/Magpie/blob/master_v0.8.1/docs/CustomScaleConfiguration.md)
> 
> Take a look at this as well [gist.github.com/hooke007/818ecc88f18e229bca743b7ae48947ad](https://gist.github.com/hooke007/818ecc88f18e229bca743b7ae48947ad)

Thank you very much! I'm experimenting a little, and I'm liking the FXAA Medium preset that Hooke made. I think I'll stick to it!
